cg/server/api.py

```diff
@@ -481,6 +481,14 @@ def get_orders():
     return make_response(response.model_dump())
 
 
+@BLUEPRINT.route("/orders/<order_id>")
+def get_order(order_id: int):
+    """Return an order."""
+    order_service = OrderService(db)
```

Comment:
Pass via config if possible

Reply:
Another PR since we have no config or context in this file.